manifests/windows/install.pp

```diff
@@ -32,7 +37,7 @@
     content => template('puppet_agent/install_puppet.bat.erb')
   }->
   exec { 'install_puppet.bat':
-    command   => "${::system32}\\cmd.exe /c start /b ${::system32}\\cmd.exe /c \"${env_temp_variable}\\install_puppet.bat\"",
+    command   => "${_cmd_location} /c start /b ${_cmd_location} /c \"${env_temp_variable}\\install_puppet.bat\"",
```

Comment:
But it does not.  On PE 3.8 i386 it resolves as sysnative.  I opened a IRB session to verify that it said file exists and that cmd.exe was executable